frontend/pages/review.tsx

```diff
@@ -106,136 +152,129 @@ function ApprovalList({ category, filter }: { category: ApprovalCategory; filter
-                  data-test={`approveButton${approvalObj.approvalType}${
-                    category === 'Upload' ? approvalObj.version?.model?.uuid : approvalObj.deployment?.uuid
-                  }`}
-                  disabled={approvalObj.status === 'Accepted'}
-                >
-                  Approve
-                </Button>
-              </Box>
+                {category === 'Deployment' && (
+                  <>
+                    <Link href={`/deployment/${approvalObj.deployment?.uuid}`} passHref legacyBehavior>
+                      <MuiLink
+                        variant='h5'
+                        sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
+                      >
+                        {approvalObj.deployment?.metadata?.highLevelDetails?.name}
+                      </MuiLink>
+                    </Link>
+                    <Typography variant='body1'>
+                      Requesting deployment of{' '}
+                      <Link href={`/model/${approvalObj.deployment?.model?.uuid}`} passHref legacyBehavior>
+                        <MuiLink
+                          sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
+                        >
+                          {approvalObj.deployment?.model?.latestVersion?.metadata?.highLevelDetails?.name}
+                        </MuiLink>
+                      </Link>
+                    </Typography>
+                    {approvalObj.deployment === undefined ||
+                      (approvalObj.deployment === null && (
+                        <Alert sx={{ mt: 2 }} severity='warning'>
+                          This deployment appears to have data missing - check with the requester to make sure it was
+                          requested correctly
+                        </Alert>
+                      ))}
+                  </>
+                )}
+              </Grid>
+              <Grid item xs={12} md sx={{ display: 'flex' }}>
+                <Box ml='auto' my='auto'>
+                  {approvalObj.status !== 'No Response' && (
+                    <Chip label={approvalObj.status} color={approvalObj.status === 'Accepted' ? 'success' : 'error'} />
+                  )}
+                </Box>
+              </Grid>
+              <Grid item sx={{ display: 'flex', width: 200 }}>
+                <Box ml='auto' my='auto'>
+                  <Button
+                    color='secondary'
+                    variant='outlined'
+                    onClick={() => changeState('Declined', approvalObj)}
+                    sx={{ mr: 1 }}
+                    disabled={approvalObj.status === 'Declined'}
+                  >
+                    Reject
+                  </Button>
+                  <DisabledElementTooltip
+                    conditions={[!canApprove() ? 'technical reviewer needs to appove this model first' : '']}
```

Comment:
See previous comment about getting the actual schema title for "technical reviewer" in case it differs in other schemas

Reply:
Resolved